src/components/comment/comment_timeline.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import { CommonProps, keysOf } from '../common';
+import classNames from 'classnames';
+import { EuiIcon } from '../icon';
+
+export type EuiCommentTimelineProps = CommonProps & {
+  /**
+   * Main icon that accompanies the comment. The default is `user` for regular comments and `dot` for update comments. To customize, pass a `string` as an `EuiIcon['type']` or any `ReactNode`.
+   */
+  timelineIcon?: ReactNode | string;
```

Comment:
```suggestion
  timelineIcon?: IconType;
```

and import `IconType` from '../icon'